src/components/core/DpChangeStateAtDate.vue

```diff
@@ -64,7 +64,7 @@
 </template>
 
 <script>
-import { formatDate, toDate } from '@demos-europe/demosplan-utils'
+import { formatDate, toDate } from '@demos-europe/demosplan-ui'
```

Comment:
Can you add pls `~~/utils` in `webpack.config.js` in PR https://github.com/demos-europe/demosplan-ui/pull/72
because there was only
```
~~/components
~~/directives
 ~~/mixins'
```
then can I Merge?

Reply:
Or after Merge both PR, make a new PR?

Reply:
i added a `~~/utils` alias here:

https://github.com/demos-europe/demosplan-ui/pull/72/files/85dde70f247008c93f78ae4be1d257692175ee1f..883664a1d481448ed6da0373c89354ded87655e4